data/json/npcs/talk_tags.json

```diff
@@ -1263,13 +1669,17 @@
       "thing right out of a scary movie",
       "<swear> thing",
       "whatever-the-fuck that is",
-      "eldritch horror"
+      "eldritch horror",
+      "monstrosity",
+      "denzien of hell",
+      "unholy being",
+      "creepy <swear> piece of <swear!>"
```

Comment:
`<swear!>` doesn't work in this context, it has things like `damn`, `damnit`, `Oh sugar!`, etc.  Maybe just throw another `<monster>` in there: `"creepy <swear> <monster>"`?

Reply:
Yeah, I can look at doing this and finding something ideal for this line.